stylus/components/forms.styl

```diff
@@ -428,6 +428,7 @@ $select
     appearance none
     background embedurl('../../assets/icons/illus/bottom-select.svg') right rem(16) center no-repeat
     background-size rem(14)
+    text-align left
```

Comment:
Why was this necessary ? Because otherwise in a context where the text-align is different (center for example), we want to reset it ?

Reply:
Because I'm using a `button` to render the fake select. And the default text align for buttons is center. But we always want the text to be aligned on the left by default when rendering something that looks like a select, no matter which underlying element is used.